packages/core/src/internal/decorators/i18n.ts

```diff
@@ -96,3 +110,45 @@ export function i18n() {
       : standardI18n(descriptor, protoOrDescriptor);
   };
 }
+
+export function getI18nValues(values: Record<string, unknown>, component: I18nElement): Record<string, unknown> {
+  if (isNilOrEmpty(values)) {
+    let returnVal = {};
+
+    if (component.hasAttribute('cds-i18n')) {
+      const stringVal = component.getAttribute('cds-i18n') + '';
+      if (isNilOrEmpty(stringVal)) {
+        returnVal = {};
+      } else {
+        try {
+          returnVal = JSON.parse(stringVal);
+        } catch {
+          console.warn('Clarity i18n: Invalid JSON passed to cds-i18n');
```

Comment:
The i18n needs to be valid JSON. That's probably the hardest thing to understand about it.

Reply:
I should use the LogService here and test against that...